flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/Edge.java

```diff
@@ -34,10 +34,10 @@
 
 	public Edge(){}
 
-	public Edge(K src, K trg, V val) {
-		this.f0 = src;
-		this.f1 = trg;
-		this.f2 = val;
+	public Edge(K source, K target, V value) {
```

Comment:
To make them consistent with naming style in other classes.
Do you suggest to revert this?